application/action/request.js

```diff
@@ -15,6 +15,10 @@ module.exports = {
 
         client = new Client(apiName);
 
+        if (bodyType === 'json-param') {
+            bodyParams = bodyParams[Object.keys(bodyParams)[0]];
```

Comment:
I hate to bring this up so late in the game, but I feel like there should be a more intuitive / less surprising way to achieve this in the configuration.

The rule, "if it's a `json-param`, it's going to be the first param in your params list," isn't something you'd expect to happen in my opinion.